### PR TITLE
Update the juju-cards project

### PR DESCRIPTION
**Pull request by anthonydillon**
Updated the juju-card project to fix the logo.

## QA
- Go to the demo
- Check out a charm
- Scroll to the card
- See it renders nicely

Fixes https://github.com/juju/juju-gui/issues/4016

### CODE REVIEW

File: gui/src/app/components/entity-details/content/content.js [hunk 1]
```diff
@@ -584,13 +584,15 @@ class EntityContent extends React.Component {
...
     const script = '<script ' +
-      'src="https://assets.ubuntu.com/v1/juju-cards-v1.6.0.js"></script>\n' +
+      'src="https://assets.ubuntu.com/v1/juju-cards-v1.7.2.js"></script>\n' +
       `<div class="juju-card" ${dataDD} data-id="${storeId}"></div>`;
     let cardElement = <div className="juju-card" data-id={storeId}></div>;
     if (ddeploy) {
       cardElement = <div className="juju-card" data-dd data-id={storeId}></div>;
     }
 
+    cardElement = <div className="juju-card" data-dd data-id="trusty/wordpress-5"></div>;
```
Comment (hatched) [inline gui/src/app/components/entity-details/content/content.js]: Every card is for wordpress? :)

**Comment by hatched** [comment]
QA, it doesn't appear to work:

Screen Shot 2019-08-30 at 11 10 19 AM

File: templates/index.html [hunk 2]
```diff
@@ -240,7 +240,15 @@
         });
       };
     </script>
-    <script src="https://assets.ubuntu.com/v1/juju-cards-v1.6.0.js"></script>
+    <script src="https://assets.ubuntu.com/v1/juju-cards-v1.7.2.js"></script>
+    <link rel="stylesheet" href="https://assets.ubuntu.com/v1/juju-cards-v1.7.2.css" />
+
+    <script>
+      setTimeout(function() {
```
Comment (hatched) [inline templates/index.html]: Why is this in a 5s setTimeout?

Is this `updateHead` required? It looks like you've hardcoded the path anyways.

Comment (anthonydillon) [reply templates/index.html]: Hmmm ... I think I need to hook this function into componentDidMount.

**Comment by anthonydillon** [comment]
Not needed now there is a redirect in place.

**Pull request closed without merging**